src/main/java/com/lunark/lunark/auth/dto/AccountSignUpDto.java

```diff
@@ -12,13 +13,32 @@
 import java.util.HashSet;
 
 public class AccountSignUpDto {
+    @PositiveOrZero
     private Long id;
+    @NotNull
```

Comment:
I think this might be covered by notempty.

Reply:
Yes, it is. Good catch.